integration-libs/s4-service/root/facade/cancel-service-order.facade.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { CancelObj } from '../model/checkout-service-details.model';
+import { Order } from '@spartacus/order/root';
+
+@Injectable({
+  providedIn: 'root',
+})
+export abstract class CancelServiceOrderFacade {
+  /**
+   * Set service schedule DateTime for the order
+   *
+   * @param scheduledAt a service date time
+   */
+  abstract cancelService(
+    orderCode: string,
+    cancelobj: CancelObj
```

Comment:
`major:` When `CancelObj` is renamed to the descriptive name, let's change names of props to corresponding ones, e.g.
```
cancellationDetails: CancellationDetails
```